pages/certificate/[tool].tsx

```diff
@@ -0,0 +1,167 @@
+            <Box mx={"15px"} maxW={"1500px"} w={"100%"}>
+                <ModuleHeader
+                    links={[
+                        {
+                            name: "Toolkit",
+                            url: `/`,
+                        },
+                        {
+                            name: data?.title || "",
+                            url: `/tool/${data?._id || ""}`,
+                        },
+                    ]}
+                />
+                <Flex flexDirection={"row"} justifyContent={"center"}>
+                    <Box
+                        mt={large ? "30px" : "20px"}
+                        fontFamily={"Inter"}
+                        w={"100%"}
+                        maxW={"800px"}
+                    >
+                        <Box fontWeight={"bold"} mb={"20px"}>
+                            Congratulations! You’ve finished the{" "}
+                            {data?.title || ""} Tool.{" "}
+                        </Box>
+                        <Box mb={"10px"}>
+                            As a testament to your effort and hard work, we’re
+                            proud to present this certificate to you.
+                        </Box>
+                        <Box w="100%">
+                            <Flex
+                                justifyContent={large ? "flex-start" : "center"}
+                            >
+                                <Image
+                                    src="/assets/certificate-preview.svg"
+                                    w="50%"
+                                    minW="300px"
+                                    maxW="100%"
+                                />
+                            </Flex>
+                        </Box>
+                        <Box mt={"10px"}>
+                            <Flex
+                                justifyContent={large ? "flex-start" : "center"}
+                            >
+                                Access your personalized certificate:
+                            </Flex>
+                        </Box>
+                        <Box
+                            mt={large ? "40px" : "20px"}
+                            mb={"10px"}
+                            w={"100%"}
+                            justifyContent={large ? "flex-start" : "center"}
+                        >
+                            <Flex
+                                flexDirection={"column"}
+                                alignItems={large ? "flex-start" : "center"}
+                            >
+                                <Flex
+                                    flexDirection={large ? "row" : "column"}
+                                    justifyContent={
+                                        large ? "flex-start" : "center"
+                                    }
+                                    mx={"-10px"}
+                                >
+                                    <Button
```

Comment:
Setting this button to loading while the data is fetching will avoid a default pdf from being generated